exit.go

```diff
@@ -176,16 +189,8 @@ func (te *TunaExit) handleSession(session *smux.Session, conn net.Conn) {
 					if in == 0 && out == 0 {
 						continue
 					}
-					service, err := te.getService(byte(i))
-					if err != nil {
-						continue
-					}
-					serviceInfo := te.config.Services[service.Name]
-					entryToExitPrice, exitToEntryPrice, err := ParsePrice(serviceInfo.Price)
-					if err != nil {
-						continue
-					}
-					totalCost += entryToExitPrice*common.Fixed64(in)/TrafficUnit + exitToEntryPrice*common.Fixed64(out)/TrafficUnit
+
+					totalCost += te.Common.entryToExitPrice*common.Fixed64(in)/TrafficUnit + te.Common.exitToEntryPrice*common.Fixed64(out)/TrafficUnit
```

Comment:
This is not correct. On the exit side, each service can charge for different price. `te.Common.entryToExitPrice` should only be used on user side, not service provider side.